config/config.js

```diff
@@ -1,26 +1,12 @@
 require('dotenv').config()
 
 module.exports = {
-    "development": {
-        "username": process.env.DB_USER,
-        "password": process.env.DB_PASSWORD,
-        "database":  process.env.DB_NAME,
-        "host": process.env.DB_HOST,
-        "port": process.env.DB_PORT,
-        "dialect": "mysql"
-    },
-    "test": {
-        "username": process.env.DB_USER,
-        "password": process.env.DB_PASSWORD,
-        "database": process.env.DB_NAME,
-        "host": process.env.DB_HOST,
-        "dialect": "mysql"
-    },
-    "production": {
-        "username": process.env.DB_USER,
-        "password": process.env.DB_PASSWORD,
-        "database": process.env.DB_NAME,
-        "host": process.env.DB_HOST,
-        "dialect": "mysql"
-    }
+    dialect: process.env.DB_DIALECT,
+    storage: process.env.DB_URI,
+
+    seederStorage: 'sequelize',
+    seederStorageTableName: 'seeds',
+
+    migrationStorage: 'sequelize',
+    migrationStorageTableName: 'migrations'
```

Comment:
por qué se hace este cambio?

Reply:
Lo coloque así porque utilize SQLite, pero después noté que no tenía que cambiar nada y solo añadir dos líneas

"development": {
        "username": process.env.DB_USER,
        "password": process.env.DB_PASSWORD,
        "database":  process.env.DB_NAME,
        "host": process.env.DB_HOST,
        "port": process.env.DB_PORT,
        "dialect": proccess.env.DB_DIALECT,
        "storage": process.env.DB_STORAGE,
    }

Y así utilizar el archivo .env con las variable